Makefile

```
@@ -219,6 +219,21 @@ fmt-check:
 .PHONY: test
 test:
 	GO111MODULE=on $(GO) test -mod=vendor -tags='sqlite sqlite_unlock_notify' $(PACKAGES)
+	@echo "\n==>\033[32m Ok\033[m\n"
+
+PHONY: test-check
+test-check:
+	@echo "Checking if tests have changed the source tree...";
+	@diff=$$(git status -s); \
+	if [ -n "$$diff" ]; then \
+		echo "make test has changed files in the source tree:"; \
+		echo "$${diff}"; \
+		echo "You should change the tests to create these files in a temporary directory."; \
+		echo "Do not simply add these files to .gitignore"; \
+		exit 1; \
+	fi;
+	@echo "==>\033[32m Ok\033[m\n";
```

These escape commands are terminal dependent (e.g. `linux`, `vt220`, etc.) and won't work the same everywhere. If you really want to highlight the text you can do something like:

```suggestion
	@echo "==> `tput setaf 2 2>/dev/null`Ok`tput sgr0 2>/dev/null`"  ; echo
```

`2>/dev/null` will hide the errors in case `tput` is not present or the `setaf` capability is not supported by the terminal.

---

Or maybe just don't log color at all 😉

---

I've removed them. The makefile already had other sections with color so I thought it was ok.